.github/workflows/rpcs3.yml

```diff
@@ -0,0 +1,171 @@
+name: Build RPCS3
+
+on:
+  push:
+    paths-ignore:
+      - '.cirrus.yml'
+      - '.azure-pipelines.yml'
+      - 'README.md'
+  pull_request:
+    paths-ignore:
+      - '.cirrus.yml'
+      - '.azure-pipelines.yml'
+      - 'README.md'
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.ref }}-${{ github.event_name }}
+  cancel-in-progress: true
+
+env:
+  BUILD_REPOSITORY_NAME: ${{ github.repository }}
+  BUILD_SOURCEBRANCHNAME: ${{ github.ref_name }}
+  BUILD_ARTIFACTSTAGINGDIRECTORY: ${{ github.workspace }}/build
+
+jobs:
+  Linux_Build:
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: ubuntu-24.04
+            docker_img: "rpcs3/rpcs3-ci-focal:1.9"
+            build_sh: "/rpcs3/.ci/build-linux.sh"
+            compiler: clang
+          - os: ubuntu-24.04
+            docker_img: "rpcs3/rpcs3-ci-focal:1.9"
+            build_sh: "/rpcs3/.ci/build-linux.sh"
+            compiler: gcc
+          - os: ubuntu-24.04-arm
+            docker_img: "rpcs3/rpcs3-ci-focal-aarch64:1.0"
+            build_sh: "/rpcs3/.ci/build-linux-aarch64.sh"
+            compiler: clang
+    runs-on: ${{ matrix.os }}
+    env:
+      CCACHE_DIR: ${{ github.workspace }}/ccache
+      CI_HAS_ARTIFACTS: true
+      DEPLOY_APPIMAGE: true
+      APPDIR: "/rpcs3/build/appdir"
+      ARTDIR: "/root/artifacts"
+      RELEASE_MESSAGE: "/rpcs3/GitHubReleaseMessage.txt"
+      COMPILER: ${{ matrix.compiler }}
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@main
+        with:
+          submodules: recursive
```

Comment:
We don't want to checkout any submodules here since we do filtered checkout of submodules in the build script.

Reply:
Removed